FBReader/src/main/java/org/geometerplus/android/fbreader/benetech/OptionsMenuHandler.java

```diff
@@ -164,10 +164,10 @@ public boolean onCreateOptionsMenuWithoutPluginActions(Menu menu){
         return onCreateOptionsMenu(menu,new LinkedList<PluginApi.ActionInfo>());
     }
 
-    public void hideMainScreenMenuOptions(Menu menu) {
-        menu.findItem(R.id.menu_item_logout_bookshare).setVisible(false);
-        menu.findItem(R.id.menu_item_login_bookshare).setVisible(false);
-        menu.findItem(R.id.menu_item_help).setVisible(false);
-        menu.findItem(R.id.menu_item_sync_with_bookshare).setVisible(false);
+    public void hideMenuOptions(Menu menu, int... ids) {
+        if(ids == null)return;
```

Comment:
Ids should be rename to menuItemIds,  and "i" inside the for statement should be renamed to menuItemId.  The caller of this method should never pass a null list, we can remove the null test.